website/www/site/static/js/section-nav.js

```diff
@@ -25,7 +25,8 @@ $(document).ready(function () {
             ACTIVE_CLASS: "active",
             EXPANDED_CLASS: "expanded",
             DESKTOP_BREAKPOINT: 1280,
-            OPEN_CLASS: "open"
+            OPEN_CLASS: "open",
+            LOCAL_HOST: "//localhost:1313"
```

Comment:
I'd prefer a solution that makes sure we use relative urls (drop the `//localhost:1313`) in the dev server. This is kind of brittle (e.g. what if you run your dev server on a different port?), and it would be on our production website.

Do you know what we'd need to modify in the build.gradle for this?